src/Infrastructure.Dapper/Repositories/CollectionRepository.cs

```diff
@@ -309,6 +312,219 @@ public async Task<ICollection<CollectionAccessSelection>> GetManyUsersByIdAsync(
         }
     }
 
+    public async Task CreateDefaultCollectionsAsync(Guid organizationId, IEnumerable<Guid> affectedOrgUserIds, string defaultCollectionName)
+    {
+        if (!affectedOrgUserIds.Any())
+        {
+            return;
+        }
+
+        using (var connection = new SqlConnection(ConnectionString))
+        {
+            connection.Open();
+            using (var transaction = connection.BeginTransaction())
```

Comment:
Rider informs me this should be `await using`, because it's an `IAsyncDisposable` interface.

⛏️ I also recommend `await using var` without brackets or indentation, just to avoid excessive indentation.

Reply:
We don't have any linting set up for `server` other than `dotnet format` which is pretty minimal. Bishop has wanted to add [MegaLinter](https://megalinter.io/latest/) for a while but I think there was some concern around the "kitchen sink" nature of that, so it hasn't progressed. I do wish we had a solution here though; feel free to follow it up in `#architecture` if you want.

In the meantime I find Rider's intellisense suggestions to be pretty good.